src/pool/Pool.t.sol

```diff
@@ -1221,6 +1231,169 @@ contract PoolTest is BCdpManagerTestBase {
+            assertTrue(! canKeepersBite(cdp));
+            assertEq(uint(dink), expectedEth);
+            assertEq(vat.gem("ETH", address(members[i])), expectedEth);
+            (uint cdpArt, uint cdpCushion, address[] memory winners, uint[] memory bite) = pool.getCdpData(cdp);
+            cdpArt; //shh
+            cdpCushion; //shh
+            winners; //shh
+            assertEq(bite[i], 26 ether);
+            assertEq(pool.rad(address(members[i])), (1000 ether - 50 ether * i - 26 ether) * RAY - 1);
+        }
+
+        // jar should get 2% from 104 * 1.1 / 130
+        assertEq(vat.gem("ETH", address(jar)), expectedInJar * 4);
+    }
+
+    // 1 DAI = 1.15 USD
+    function testVeryHighDaiRateInUsd() public {
+        members[0].doDeposit(pool, 1000 ether * RAY);
+        members[1].doDeposit(pool, 950 ether * RAY);
+        members[2].doDeposit(pool, 900 ether * RAY);
+        members[3].doDeposit(pool, 850 ether * RAY);
+
+        uint cdp = openCdp(1 ether, 104 ether); // 1 eth, 110 dai
+
+        // set next price to 150, which means a cushion of 10 dai is expected
+        osm.setPrice(150 * 1e18); // 1 ETH = 150 DAI
+
+        members[0].doTopup(pool, cdp);
+
+        pipETH.poke(bytes32(uint(150 * 1e18)));
+        spotter.poke("ETH");
+        realPrice.set("ETH", 130 * 1e18);
+
+        // Update DAI to USD rate
+        FakeDaiToUsdPriceFeed dai2usd = FakeDaiToUsdPriceFeed(address(pool.dai2usd()));
+        dai2usd.setPrice(1.15 * 1e18); // 1 DAI = 1.15 USD
+
+        //uint ethBefore = vat.gem("ETH", address(members[0]));
+        this.file(address(cat), "ETH", "chop", WAD + WAD/10);
+        pool.setProfitParams(93, 100); // 7% goes to jar
+        // for 26 ether we expect 26/130 * 1.1 = 28.6/130, from which 93% goes to member
+        uint _100Percent = 286 ether / (130 * 10);
+        // _100Percent * 93% * 1.15
+        uint expectedEth = _100Percent * uint(93) * 115 / (100 * 100);
+
+        assertTrue(_100Percent < expectedEth);
+        expectedEth = (expectedEth > _100Percent) ? _100Percent : expectedEth;
+        uint expectedInJar = 0;
+
+        for(uint i = 0 ; i < 4 ; i++) {
+            assertTrue(! canKeepersBite(cdp));
+            uint dink = members[i].doPoolBite(pool, cdp, 26 ether, expectedEth);
+            assertTrue(! canKeepersBite(cdp));
+            assertEq(uint(dink), expectedEth);
+            assertEq(vat.gem("ETH", address(members[i])), expectedEth);
+            (uint cdpArt, uint cdpCushion, address[] memory winners, uint[] memory bite) = pool.getCdpData(cdp);
+            cdpArt; //shh
+            cdpCushion; //shh
+            winners; //shh
+            assertEq(bite[i], 26 ether);
+            assertEq(pool.rad(address(members[i])), (1000 ether - 50 ether * i - 26 ether) * RAY - 1);
+        }
+
+        // jar should get 2% from 104 * 1.1 / 130
+        assertEq(vat.gem("ETH", address(jar)), expectedInJar);
+    }
+
```

Comment:
i think very low rate should be ok. but nevertheless could you please add a test for 1 DAI = 0.8 USD?